core/rest/pom.xml

```diff
@@ -417,29 +406,19 @@
   	<compileSource>1.6</compileSource>
     
   	<!-- Dependencies -->
-    <hadoop.version>${env.HADOOP_DEP_VER}</hadoop.version>
+    <hadoop.version>2.6.0</hadoop.version>
```

Comment:
Do we want to hard-code this? If my memory serves, this was parameterized only recently.